apps/asap-server/src/utils/validate-user.ts

```diff
@@ -23,18 +19,7 @@ export default async function validateUser(
     throw Boom.unauthorized();
   }
 
-  const [error, res] = await Intercept(
-    got(`https://${authConfig.domain}/userinfo`, {
-      headers: {
-        Authorization: `Bearer ${token}`,
-        'Content-Type': 'application/json',
-      },
-    }).json<auth0.UserInfo>(),
-  );
-
-  if (error) {
-    throw Boom.forbidden('Forbidden', { error });
-  }
-
-  return res;
+  return decodeToken(token).catch(() => {
+    throw Boom.forbidden();
```

Comment:
Unauthorized is correct if the token is invalid - forbidden would be the token is valid but has insufficient privileges for the operation